### PR TITLE
Add documentation for NFC & QR Tags

### PR DESCRIPTION
**Pull request by zacwest**

### CODE REVIEW

File: docs/integrations/universal-links.md [hunk 1]
```diff
@@ -1,9 +1,43 @@
...
 ---
 
-![iOS](/assets/apple.svg)
+Support for these features is active under development and may not be fully released yet.
+
+| Feature   | ![iOS](/assets/apple.svg) iOS | ![android](/assets/android.svg) Android |
+| --------- | ----------------------------- | --------------------------------------- |
+| NFC Tags  | 2020.5                        | Coming Soon                             |
+| QR Tags   | 2020.5                        | Coming Soon                             |
+| Old-Style | 2.0                           | Not Supported                           |
```
Comment (TomBrien) [inline docs/integrations/universal-links.md:12]:
```suggestion
| Old-Style |  2019.1                      | Not Supported                           |
```

Comment (TomBrien) [inline docs/integrations/universal-links.md:12]: @dshokouhi do we need to split the Android column into Full & Minimal?

Comment (dshokouhi) [reply docs/integrations/universal-links.md:12]: @TomBrien no need to, it will support both flavors based on the PR submitted

**Comment by TomBrien** [comment]
Looks good here.

**Pull request merged**